### DISCUSSION TITLE
laggy input typing [Oct 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4310] [Discord Threads]
> 🙋‍♂️ **LeewardB**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### can u share more? is it singleLineText or LongText? how many records you hv and also the project info  
#  
> 🙋‍♂️ **LeewardB**  
#### i found a related issue in the tracker -- https://github.com/nocodb/nocodb/issues/4103  
#  
> 🙋‍♂️ **LeewardB**  
#### and maybe even a fix already merged to develop? https://github.com/nocodb/nocodb/pull/4088  
#  
> 🙋‍♂️ **LeewardB**  
#### it's happening in a brand new project, in a brand new nocodb, with only 1 row, in a short text box  
#  
> 🙋‍♂️ **LeewardB**  
#### in the automatic "title" field, if that matters  
#  
> 🙋‍♂️ **wkw**  
#### you may verify it with this tag   
####    
#### ```   
#### docker run -d -p 8888:8080 nocodb/nocodb-timely:0.98.1-pr-4088-20221015-0722   
#### ```  
#  
> 🙋‍♂️ **wkw**  
#### we havent released this bug fix  
#  
> 🙋‍♂️ **LeewardB**  
#### cool thanks 🙂  
#  
> 🙋‍♂️ **LeewardB**  
#### I got it working 🙂 #4088 was not actually the correct issue (i grabbed the wrong link from the #4103 discussion), but i found the fix in `4118-20221017-1023` and verified the new code works appropriately  
#  
> 🙋‍♂️ **LeewardB**  
#### huge improvement, thanks for the quick help, i wouldn't have noticed that there are easy docker tags for each PR!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1033829512347144273)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
